Sources/IO/Misc/OBJReader/index.js

```diff
@@ -195,12 +195,12 @@ function vtkOBJReader(publicAPI, model) {
     model.compression = option.compression;
 
     // Fetch metadata
-    return publicAPI.loadData();
+    return publicAPI.loadData({progressCallback: option.progressCallback});
   };
 
   // Fetch the actual data arrays
-  publicAPI.loadData = () => {
-    const promise = fetchData(model.url);
+  publicAPI.loadData = (option = {}) => {
+    const promise = fetchData(model.url, {progressCallback: option.progressCallback});
```

Comment:
Based on your previous comment you right, `fetchText` should probably take some options.
So the reader just forward those options over.

For the doc, we usually ask the user to write a `api.md` next to the class definition.